contrib/builder/arm/deb/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ioft/armhf-debian:jessie
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libsqlite3-dev libsystemd-journal-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.5.1
+RUN curl -fSL "https://github.com/hypriot/golang-armbuilds/releases/download/v${GO_VERSION}/go${GO_VERSION}.linux-armv7.tar.gz" | tar xzC /usr/local
```

Comment:
Possibly the same here, I know there's been discussion around using binaries in stead of building from source. Not sure what our options are here, so just commenting :smile: